src/humanize/number.py

```diff
@@ -172,16 +178,27 @@ def intword(value, format="%.1f"):
     except (TypeError, ValueError):
         return value
 
-    if value < powers[0]:
+    if conversions is None:
+        l_powers, l_human_powers = powers, human_powers
```

Comment:
What does the `l_` prefix mean?

Reply:
`l_` as in `local`. 

I might have done it the other way around, by renaming the default ones, but tests were specifically looking for `powers` and `human_powers` in the body of `number`. I'll change this and adjust the tests as well.